tools/ddsrouter_tool/test/CMakeLists.txt

```diff
@@ -14,3 +14,7 @@
 
 # Add subdirectory with tests
 add_subdirectory(application)
+
```

Comment:
I don't think we need another extra package just for testing which does nothing in case the CMake option is not defined

Reply:
I just see it confusing that this test could be compiled and executed without its subpackage or any other subpackage in the project.
I think it would make more sense in a different independent subpackage.